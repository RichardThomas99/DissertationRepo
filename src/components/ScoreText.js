import React from 'react';
import'../App.css';

function ScoreText()
{

  return(
    <div>
    <h4>Score Element to presented in table component below</h4>

    <p>Score to be out of 100 and to indexed so the better the listing the closer to 100 it is.</p>
    </div>
  );
}

export default ScoreText;
