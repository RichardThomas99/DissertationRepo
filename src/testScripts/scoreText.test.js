import React from 'react';
import ReactDOM from 'react-dom';
import ScoreText from '../components/ScoreText';
import renderer from 'react-test-renderer'

it("ScoreText Render",() => {
  const scoreTextComponent = renderer.create(
    <ScoreText/>
  ).toJSON();
  expect(scoreTextComponent).toMatchSnapshot();
});
