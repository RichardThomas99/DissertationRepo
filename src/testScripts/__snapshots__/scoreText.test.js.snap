// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScoreText Render 1`] = `
<div>
  <h4>
    Score Element to presented in table component below
  </h4>
  <p>
    Score to be out of 100 and to indexed so the better the listing the closer to 100 it is.
  </p>
</div>
`;
